### PR TITLE
k-bucket Update to v5.0.0 / esModuleInterop issue on client integration Fix

### LINKED ISSUE #87: esModuleInterop issue on client integration
Have now locally linked the `devp2p` library to the client library to fix the issues from https://github.com/ethereumjs/ethereumjs-client/issues/143

I will now fix the issues like 

```shell
Module '"/ethereumjs-devp2p/node_modules/@types/lru-cache/index"' can only be default-imported using the 'esModuleInterop' flag
```

by switching to `import =` syntax as suggested [here](https://github.com/microsoft/TypeScript/issues/27293#issuecomment-549047461) and push later the day.

Let me know if this fix is not adequate, solution is directly taken from the `TypeScript` GitHub (might be wrong context though) and `esModuleInterop` is activated both on the client and the devp2p library.

### PR DESCRIPTION
**Pull request by holgerd77**
Fixes #87 and continues #79

### CODE REVIEW

File: src/dpt/kbucket.ts [hunk 1]
```diff
@@ -59,12 +59,12 @@ export class KBucket extends EventEmitter {
     return null
   }
 
-  getAll() {
+  getAll(): any {
```
Comment (jochem-brouwer) [inline src/dpt/kbucket.ts:62]: Nit: can make this more typesafe? Should return `Array<T>`?

**Comment by jochem-brouwer** [comment]
LGTM, one nit comment

**Review by jochem-brouwer — APPROVED** [review]

**Comment by holgerd77** [comment]
Wonder if this already fixed a super-significant bug along the way, I am seeing numbers of peers on the client run I've actually never seen before? 🤔

image

The change in `closest()` would be the only candidate for this I would say.

**Comment by holgerd77** [comment]
@jochem-brouwer thanks for the quick look, have updated!

**Comment by holgerd77** [comment]
Or eventually it's also the `k-bucket` update, thanks @tomonari-t for having prepared this!

**Comment by jochem-brouwer** [comment]
I actually do recall seeing that amount of peers before @holgerd77, but that might be because I bootstrapped it to my own node?

It could indeed the this `closest` thing 😄  Thanks for the type update.

**Review by jochem-brouwer — APPROVED** [review]

**Pull request merged**